rav1e_js/www/public/index.html

```diff
@@ -20,6 +20,16 @@
 
 <body>
   <noscript>You need to enable JavaScript to run this app.</noscript>
+
+  <img src="octocat.png" alt="Octocat" id="octocat" width="200px" height="200px"></img>
+  <img src="ferris.png" alt="Ferris" id="ferris" width="200px" height="200px"></img>
```

Comment:
Yeaah, not that easy. The problem is, that both pictures need to have the same dimensions, to send them to the same encoder.

Solutions I can think of rn:
* get two pictures of the same dimensions
* wrap the smaller picture with black/transparent/white pixels to get the same dimesions as the other one
    --> is there a way to do sth like that already in rav1e?

Reply:
getting two dummy pics of the same dimension will be cool, 
I did not understand the need for two pictures at the first place.

Reply:
Actually it is still from a prior demo where I had two pictures and were sending both to the encoder. But since the video is the star now I will remove them.